### PR TITLE
ready

### PR DESCRIPTION
**Pull request by denenergi**
[DEMO LINK](https://denenergi.github.io/react_tabs-with-router/)

### CODE REVIEW

**Comment by HennadiiKupr** [comment]
In the demo, you don't have any focus effects in the navigation bar

File: src/components/TabsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+import { TabLink } from './TabLink';
+
+interface Props {
+  tabs: Tab[]
+}
+
+export const TabsPage:React.FC<Props> = ({ tabs }) => {
+  // const [isActive, setIsActive] = useState('');
```
Comment (HennadiiKupr) [inline src/components/TabsPage.tsx:11]: you need to delete unnecessary comments

**Review by HennadiiKupr — CHANGES_REQUESTED** [review]

File: src/components/TabLink.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+interface Props {
+  tab: Tab;
+  // isActive: string;
+  // setIsActive: (id: string) => void;
+  setTabText: (text: string) => void;
```
Comment (ivan-kadykalo) [inline src/components/TabLink.tsx:10]: Delete comments

**Review by ivan-kadykalo — APPROVED** [review]
🚀

**Pull request closed without merging**